### PR TITLE
Update Squeex collab video

### PR DESCRIPTION
**Pull request by MattIPv4**
## Describe your changes

New produced video from the Squuex collab, replacing the current VoD.

## Notes for testing your change

N/A

### CODE REVIEW

**Review by pjeweb — APPROVED** [review]

File: apps/website/src/pages/collaborations.tsx [hunk 1]
```diff
@@ -50,7 +50,8 @@ const collaborations = {
     name: "Squeex",
     link: "https://www.twitch.tv/squeex",
     date: new Date("2023-05-19"),
-    videoId: "dCV8b8LcG-0",
+    videoId: "oUJxCDGQDlY ",
```
Comment (MattIPv4) [inline apps/website/src/pages/collaborations.tsx:53]:
```suggestion
    videoId: "oUJxCDGQDlY",
```

**Comment by MattIPv4** [comment]
Dunno how I missed that space 🙃

**Review by pjeweb — APPROVED** [review]
oh

**Comment by MattIPv4** [comment]
(Re-triggering Actions, 'cause https://www.githubstatus.com/incidents/rk1918knvr0w broke them)

**Pull request merged**